### PR TITLE
add DDP CPU example

### PR DESCRIPTION
**Pull request by KeitaW**
*Issue #, if available:*

*Description of changes:*

We need a simple ddp example to showcase distributed training on CPU for workshop studio users. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by verdimrc — APPROVED** [review]

**Comment by perifaws** [comment]
Add `pytorch` to the directory name so we know which framework is used. DDP is a parallelization method.

File: 3.test_cases/16.cpu-ddp/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## 2. Submit training job
+
+Submit DDP training job with:
+
+```bash
+sbatch 1.train.sbatch
+```
+
+Output of the training job can be found in `logs` directory.
```
Comment (perifaws) [inline 3.test_cases/16.cpu-ddp/README.md:23]: Add an example of what a user should see (at least an excerpt)

**Comment by KeitaW** [comment]
> Add `pytorch` to the directory name so we know which framework is used. DDP is a parallelization method.

Thanks! Changed test case name to `16.pytorch-cpu-ddp`.
https://github.com/aws-samples/awsome-distributed-training/pull/212/commits/a2981f014aaf760d50e83076572c7881d2e25cb1

File: 3.test_cases/16.cpu-ddp/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# PyTorch DDP on CPU <!-- omit in toc -->
+
+This test case is intended to provide simplest possible distributed training example on CPU using [PyTorch DDP](https://pytorch.org/tutorials/beginner/ddp_series_theory.html).
```
[the remark this answers is not in the document]
Comment (KeitaW) [reply 3.test_cases/16.cpu-ddp/README.md]: Fixed in https://github.com/aws-samples/awsome-distributed-training/pull/212/commits/8405a030b186ca2b61f6ee8d7d91ba36a372cd17

File: 3.test_cases/16.cpu-ddp/README.md (hunk 1 above)
Comment (KeitaW) [reply 3.test_cases/16.cpu-ddp/README.md]: Fixed in https://github.com/aws-samples/awsome-distributed-training/pull/212/commits/8405a030b186ca2b61f6ee8d7d91ba36a372cd17

**Comment by perifaws** [comment]
Thanks Keita!

**Pull request merged**